src/scripts/refgene2protein.pl

```diff
@@ -78,37 +97,114 @@ sub generate_refgene2protein {
   }
   die unless @infiles;
 
+  my $preprocess = $FLAGS{"single-gbff"} ? 0 : 1;
+
+  if ($preprocess) {
+    # parser is extremely slow, so digest input to just refGene records.
+    # even parsing the ~50k NM_ accessions is very slow but MUCH
+    # faster than having to parse everything.
+    my $md5 = md5_hex(map {$_, -s $_} @infiles);
+    my $f_cooked = sprintf "refgene.%s.txt.gz", $md5;
+    my $f_split_template = sprintf "refgene_split.%s.txt.%%03d.gz", $md5;
+
+    unless (-s $f_cooked) {
+      my $wf = new WorkingFile($f_cooked, "-compress" => "gz");
+      my $fh_out = $wf->output_filehandle();
+
+      my $wf_chunk;
+      my $fh_chunk;
+      my $chunk_num = 0;
+      my $nm_count = 0;
+      my $chunk_size = 1000;
+
+      foreach my $infile (@infiles) {
+	printf STDERR "preprocessing %s...\n", $infile;
```

Comment:
Looks like indentation gets messed up here.  Perhaps a mix of tabs and spaces?

Reply:
The indenting looks fine in my editor (emacs) which I believe uses a mix of both by default (tabs if the indent level is divisible by the tab length, which I believe is default 8).  I think it's possible to change the behavior to have every indent use a tab character but this looks much worse with e.g. the "less" pager.  Changing to that layout would I think also make every line in the file look different (is there some kind of "diff -w" option here?).